web/themes/custom/news/src/styles/sidebar-left.scss

```diff
@@ -51,11 +51,11 @@
 
         .views-field-title {
           position: absolute;
-          top: 50%;
+          bottom: 20%;
           padding: 0 10px;
           a {
             color: #fff;
-            font-size: 17px;
+            font-size: 1vw;
```

Comment:
Is it even readable on mobile?

Reply:
this block disappears for mobile,
but i need some advice on this block, so when you are free...